LEAF_Request_Portal/admin/templates/view_form_library.tpl

```diff
@@ -33,15 +33,15 @@ function showPreview(recordID) {
     preview.load(recordID, 1, 0);
 
     $('#btn_download').one('click', function() {
-    	$.ajax({
-    		type: 'POST',
-    		url: 'ajaxIndex.php?a=importForm',
-    		data: { formPacket: JSON.stringify(preview.getRawForm()),
-    			    formLibraryID: recordID},
-    		success: function() {
-    			window.location = '?a=form&referFormLibraryID=' + recordID;
-    		}
-    	});
+        $.ajax({
```

Comment:
Needs a fail catch

Reply:
I would be curious how this works since error is removed according to the docs in jquery 3 which is what we are using from what I can see. https://api.jquery.com/Jquery.ajax/

Reply:
The method on the ajax object is removed and replaced with fail.  eg ajax({options}).done().fail().always()
The config option in the setup is still error.  ajax({success, error})
This isn't confusing at all and is why there have been fails sneaking into the config (where it gracefully does nothing).